packages/backend/src/graphql/custom-resolvers/table-metadata.ts

```diff
@@ -10,6 +12,25 @@ const columns: TableMetadataResolver['columns'] = async (parent) => {
   return columns
 }
 
+const collaborators: TableMetadataResolver['collaborators'] = async (
+  parent,
+) => {
+  const collaborators = await parent
+    .$relatedQuery('collaborators')
+    .select('email', 'role')
+    .orderBy('role', 'desc')
```

Comment:
question: actually is this `orderBy` necessary if you are sorting by the role and email using lodash next

Reply:
ure right! thanks will remove